src/Common/src/Interop/ComCtl32/Interop.TvItemSelectionChangeCause.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class ComCtl32
+    {
+        public enum TvItemSelectionChangeCause : uint
```

Comment:
```suggestion
        public enum TrewViewItemSelectionChangeCause : uint
```

Ditto for other `TvXxxx` types